Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm

```diff
@@ -173,7 +173,7 @@ String resolvePathForSandboxExtension(StringView path)
 {
     String resolvedPath = stringByResolvingSymlinksInPath(path);
     if (resolvedPath.isNull()) {
-        LOG_ERROR("Could not create a valid file system representation for the string '%s' of length %lu", resolvedPath.utf8().data(), resolvedPath.length());
+        RELEASE_LOG_ERROR(Sandbox, "Could not create a valid file system representation for the string '%s' of length %lu", resolvedPath.utf8().data(), resolvedPath.length());
```

Comment:
Weird that this isn't compiling. How did it used to work?

Reply:
That is a good question, maybe the compiler isn't validating the format string when using LOG_ERROR?

Thanks for reviewing!